docs/pages/workflow/expo-cli.mdx

```diff
@@ -45,6 +45,14 @@ You can run any command with the `--help` or `-h` flag to learn more about it:
 
 <Terminal cmd={['$ npx expo login -h']} />
 
+## Installation
+
+Expo CLI is included in the `expo` package. You can install it with npm or yarn:
+
+<Terminal cmd={['$ yarn add expo']} />
+
+> Projects that are not using [Expo Prebuild](#prebuild) (Bare workflow), will need to perform additional setup to ensure all custom Expo bundling features work: [Metro: Bare workflow setup](/versions/latest/config/metro#bare-workflow-setup).
```

Comment:
The way this is worded is a bit ambiguous, specifically the part that says, "Projects that are not using Expo Prebuild (Bare workflow)..."

1. (Projects that are not using Expo Prebuild)(which are Bare workflow projects)
2. Projects that are not using (Expo Prebuild/Bare workflow)

(1) is I think what you are looking to communicate. Suggestion that reads less ambiguously:

> Projects that are not using Expo Prebuild (sometimes called Bare workflow projects) will need to ...

(this also removes the extra comma before "will")